src/components/simulations/histogram_with_stats/index.js

```diff
@@ -0,0 +1,99 @@
+import React, {Component} from 'react'
+import {DistributionSummary} from 'gComponents/distributions/summary/index'
+import Histogram from 'gComponents/simulations/histogram/index'
+import {percentile} from 'lib/dataAnalysis'
+import $ from 'jquery'
+import './style.css'
+
+const MARGIN_LEFT = 10
+
+const percentages = (values, perc) => {
+  return perc.map(e => { return {percentage: e, value: percentile(values, values.length, e)} })
+}
+
+const findPercentile = (values, value) => {
```

Comment:
I think you can just use the `cutoff` function in `src/lib/dataAnalysis.js` to find the index.